packages/react/src/components/Item/index.js

```diff
@@ -35,7 +36,7 @@ const Item = React.forwardRef(
     const isMini = size === SIZES.MINI;
 
     const containerProps = onClick && {
-      onClick: safeInvoke(onClick),
+      onClick,
```

Comment:
Why did you remove the `safeInvoke` fn?

Reply:
I didn't have an understanding why, but this line was crashing when `Item` was used with `Dropdown` with `react-select` :/
However, existence of `onClick` is checked anyways in L37 right?

Reply:
Yep good catch 👍